### PR TITLE
Added a getting started guide for Synapse[skip ci]

### PR DESCRIPTION
**Pull request by viadea**
Added a getting started guide for Synapse.

Basically I plan to link to Azure Synapse Docs which has pretty detailed quickstart guide docs.
In the future, we only need to update the rapids spark jar version here.

### CODE REVIEW

File: docs/get-started/getting-started-azure-synapse-analytics.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+parent: Getting-Started
+---
+# Getting started with RAPIDS Accelerator on Azure Synapse Analytics
+ [Azure Synapse](https://docs.microsoft.com/en-us/azure/synapse-analytics/) is a limitless analytics service that brings
+ together enterprise data warehousing and Big Data analytics.
+
+Synapse now offers the ability to create Apache Spark pools that use GPUs on the backend to run your Spark workloads on 
+GPUs for accelerated processing. This is called as 
+[GPU-accelerated Apache Spark pools](https://docs.microsoft.com/en-us/azure/synapse-analytics/spark/apache-spark-gpu-concept).
+Currently it ships with the RAPIDS Accelerator for Apache Spark version 21.10 jars.
```
Comment (tgravescs) [inline docs/get-started/getting-started-azure-synapse-analytics.md:14]: I would remove "jars" from the end

Comment (viadea) [reply docs/get-started/getting-started-azure-synapse-analytics.md]: Removed.

File: docs/FAQ.md [hunk 2]
```diff
@@ -22,6 +22,7 @@ The RAPIDS Accelerator for Apache Spark officially supports:
 - [AWS EMR 6.2+](get-started/getting-started-aws-emr.md)
 - [Databricks Runtime 7.3, 9.1](get-started/getting-started-databricks.md)
 - [Google Cloud Dataproc 2.0](get-started/getting-started-gcp.md)
+- [Azure Synapse](get-started/getting-started-azure-synapse-analytics)
```
Comment (jlowe) [inline docs/FAQ.md:25]:
```suggestion
- [Azure Synapse](get-started/getting-started-azure-synapse-analytics.md)
```

**Review by jlowe — DISMISSED** [review]

**Review by tgravescs — APPROVED** [review]

**Comment by pxLi** [comment]
build

File: docs/download.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ Software Requirements:
 
 	CUDA & NVIDIA Drivers*: 11.0-11.4 & v450.80.02+
 
-	Apache Spark 3.0.1, 3.0.2, 3.0.3, 3.1.1, 3.1.2, 3.2.0, Cloudera CDP 7.1.6, 7.1.7, Databricks 7.3 ML LTS or 8.2 ML Runtime, and GCP Dataproc 2.0
+	Apache Spark 3.0.1, 3.0.2, 3.0.3, 3.1.1, 3.1.2, 3.2.0, Cloudera CDP 7.1.6, 7.1.7, Databricks 7.3 ML LTS or 8.2 ML Runtime, GCP Dataproc 2.0 and Azure Synapse
```
Comment (gerashegalov) [inline docs/download.md:34]:
```suggestion
	Apache Spark 3.0.1, 3.0.2, 3.0.3, 3.1.1, 3.1.2, 3.2.0, Cloudera CDP 7.1.6, 7.1.7, Databricks 7.3 ML LTS or 8.2 ML Runtime, GCP Dataproc 2.0, and Azure Synapse
```

Comment (gerashegalov) [reply docs/download.md:34]: This build list has gotten too long, maybe itemize one per line?
```
One of supported Spark versions:
- Apache Spark 3.0.1
- ...
```

File: docs/get-started/getting-started-azure-synapse-analytics.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+layout: page
+title: Azure Synapse Analytics
+nav_order: 5
+parent: Getting-Started
+---
+# Getting started with RAPIDS Accelerator on Azure Synapse Analytics
+ [Azure Synapse](https://docs.microsoft.com/en-us/azure/synapse-analytics/) is a limitless analytics service that brings
```
Comment (gerashegalov) [inline docs/get-started/getting-started-azure-synapse-analytics.md:8]: maybe defer the superlatives such as "limitless" to the documentation hosted externally?

Comment (viadea) [reply docs/get-started/getting-started-azure-synapse-analytics.md]: Removed "limitless". @gerashegalov

**Review by gerashegalov — APPROVED** [review]

**Comment by gerashegalov** [comment]
build

File: docs/get-started/getting-started-azure-synapse-analytics.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+---
+layout: page
+title: Azure Synapse Analytics
+nav_order: 5
+parent: Getting-Started
+---
+# Getting started with RAPIDS Accelerator on Azure Synapse Analytics
+ [Azure Synapse](https://docs.microsoft.com/en-us/azure/synapse-analytics/) is an analytics service that brings
```
Comment (Niharikadutta) [inline docs/get-started/getting-started-azure-synapse-analytics.md:8]: Let's change this to `Azure Synapse Analytics` is an analytics service...

File: docs/get-started/getting-started-azure-synapse-analytics.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+ together enterprise data warehousing and Big Data analytics.
+
+Synapse now offers the ability to create Apache Spark pools that use GPUs on the backend to run your Spark workloads on 
+GPUs for accelerated processing. This is called as
```
Comment (Niharikadutta) [inline docs/get-started/getting-started-azure-synapse-analytics.md:12]: These are called GPU-accelerated Apache Spark pools.

File: docs/get-started/getting-started-azure-synapse-analytics.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+# Getting started with RAPIDS Accelerator on Azure Synapse Analytics
+ [Azure Synapse](https://docs.microsoft.com/en-us/azure/synapse-analytics/) is an analytics service that brings
+ together enterprise data warehousing and Big Data analytics.
+
+Synapse now offers the ability to create Apache Spark pools that use GPUs on the backend to run your Spark workloads on 
+GPUs for accelerated processing. This is called as 
+[GPU-accelerated Apache Spark pools](https://docs.microsoft.com/en-us/azure/synapse-analytics/spark/apache-spark-gpu-concept).
+Currently it ships with the RAPIDS Accelerator for Apache Spark version 21.10.
+
+Please follow below 2 quickstart guides:
```
Comment (Niharikadutta) [inline docs/get-started/getting-started-azure-synapse-analytics.md:16]: Please follow the below quickstart guides to learn how to create and use GPU pools in Azure Synapse:

**Comment by viadea** [comment]
@Niharikadutta  This PR is already merged. I created a new [PR](https://github.com/NVIDIA/spark-rapids/pull/4199) based on your suggestion.

**Pull request merged**